runtime/doc/builtin.txt

```diff
@@ -181,13 +248,22 @@ assert_equal({expected}, {actual} [, {msg}])                    *assert_equal()*
 <		Will add the following to |v:errors|:
 			test.vim line 12: baz: Expected 'foo' but got 'bar' ~
 
+                Parameters:
+                  • expected (`any`)
+                  • actual (`any`)
+                  • msg (`any?`)
+
+                Return: (`0|1`)
```

Comment:
Again for consistency, I would format this as:

```suggestion
                Return: ~
                    (`0|1`)
```

It's a bit wasteful for on vertical space for most of this file, but it's easier to read for returns with a description and has better highlighting.

![image](https://github.com/user-attachments/assets/141d2dc8-39ef-4388-8fd1-48daad94d11e)